mobile-widgets/qml/DownloadFromDiveComputer.qml

```diff
@@ -34,7 +34,7 @@ Kirigami.Page {
 		deviceData.deviceId : 0
 		deviceData.diveId : 0
 		deviceData.saveDump : false
-		deviceData.saveLog : false
+		deviceData.saveLog : true
```

Comment:
since this is no longer optional, why not simply remove that parameter?

Reply:
A good idea, will update that.

Reply:
on second thought (and looking in code), bad idea, deviceData is shared with the desktop version, so I would need to introduce a number of #ifdef MOBILE. I think this solution is cleaner.

Reply:
And I have tested both desktop and mobile on Mac, which works.

Reply:
OK, so my review was utter nonsense.
Glad I'm able to provide value here :-(

Reply:
no no, you made me think and argue for my case, that is not at all a bad idea :-)